platform/core/call.js

```diff
@@ -3,13 +3,14 @@ const registry = require('./registry');
 
 
 class Call extends base.node.Node {
-  constructor(path) {
-    super(registry.signature(path));
+  constructor(path, method = 'get') {
```

Comment:
wait. so lets assume I have a node on path `/x` with method `post`. then a `Call` to invoke the node that does not specify the method actually would not be able to find the node, right? if that is the case, this would break backwards compatibility as it would break existing codes on the platform, and would need to be fixed.

Reply:
Fixed